YesGraph/yesgraph-superwidget
# Misc fixes

### What’s this PR do?
- Makes a bugfix to support jQuery 1.7 [[ln.665]](https://github.com/YesGraph/yesgraph-superwidget/pull/23/files#diff-6d2a12d760bbd72a115bede316ba4e18R665)
- Fixes the error whitelist for Sentry, so we won't catch other people's errors [[ln. 92]](https://github.com/YesGraph/yesgraph-superwidget/pull/23/files#diff-1533003f6889d8c764c0e592f5d703f4R92)
- Improves some of the error messages [[ln. 572]](https://github.com/YesGraph/yesgraph-superwidget/pull/23/files#diff-6d2a12d760bbd72a115bede316ba4e18L572)
- Removes the photo saving (it will be added back in a server-side PR). [[ln. 1079]](https://github.com/YesGraph/yesgraph-superwidget/pull/23/files#diff-2270d97c854398a2bc97e6f0ab3f1743L1079)
- Adds a flag to optionally remove Sentry (requested by a [customer](https://app.asana.com/0/59202558034519/159437841961249)) [[ln. 234]](https://github.com/YesGraph/yesgraph-superwidget/pull/23/files#diff-f2a29f8e395759514945b401d1de0e3cR234)
- Removes horizontal scrollbars on the contacts modal [[ln. 369]](https://github.com/YesGraph/yesgraph-superwidget/pull/23/files#diff-6d2a12d760bbd72a115bede316ba4e18L369)
- Adds a warning message for people using unsupported versions of jQuery (<1.7) [[ln. 62]](https://github.com/YesGraph/yesgraph-superwidget/pull/23/files#diff-1533003f6889d8c764c0e592f5d703f4R62)
- Adds a couple more tests!

#### How should this be manually tested?
In src/demo_widget.html, change line 16 to use jQuery 1.7.2:
[https://code.jquery.com/jquery-**2.1.1**.min.js](https://code.jquery.com/jquery-2.1.1.min.js) --> [https://code.jquery.com/jquery-**1.7.2**.min.js](https://code.jquery.com/jquery-1.7.2.min.js)

Then look at it from the browser, and it should load the widget ☺️

### What are the relevant tickets?
jQuery 1.7 bugfix: https://app.asana.com/0/59202558034519/159437841961247
Whitelisting errors: https://app.asana.com/0/59202558034519/159217416429816
Error messages: https://app.asana.com/0/59202558034519/159217416429811
Photo saving: https://app.asana.com/0/59202558034519/159217416429838
Removing Sentry: https://app.asana.com/0/59202558034519/159437841961249
Horizontal Scrollbars: https://app.asana.com/0/59202558034519/159826558648553

### Screenshots

**With Scrollbars (before):**
<img width="597" alt="screen shot 2016-07-26 at 10 40 49 am" src="https://cloud.githubusercontent.com/assets/10701968/17148848/9d85cd0c-531d-11e6-8575-39f233dbbce3.png">


**No Scrollbars (after):**
<img width="601" alt="screen shot 2016-07-26 at 10 41 20 am" src="https://cloud.githubusercontent.com/assets/10701968/17148864/a0933322-531d-11e6-81bb-4b32c8895a39.png">
